examples/controlnet/train_controlnet_flax.py

```diff
@@ -1020,10 +1047,19 @@ def cumul_grad_step(grad_idx, loss_grad_rng):
         )
         # train
         for batch in train_dataloader:
+
+            if args.profile_steps and global_step == 1:
+                train_metric['loss'].block_until_ready()
```

Comment:
Should this be `train_metrics`?

Reply:
No, this is indeed `train_metric` (note that `train_metrics` is a list). Initialized it before the loop.

(I didn't notice the failing `make quality` check, my bad!)